lib/HTTP/Headers.pm

```diff
@@ -405,6 +405,20 @@ sub content_is_xml {
     return 0;
 }
 
+sub content_is_json {
+    my $ct = shift->content_type;
+    # text/json is not standard but still used by various servers.
+    # No issue including it as well.
+    return $ct eq 'application/json' || $ct eq 'text/json';
```

Comment:
would you please broaden this criteria, so it includes `application/*+json` too

Reply:
Will do.